backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPBankAccountDAO.java

```diff
@@ -138,9 +138,9 @@ else if (query.getInvoiceId() != null)
 							.addEqualsFilter(I_C_Invoice.COLUMNNAME_C_Invoice_ID, query.getInvoiceId())
 							.create());
 		}
-		if (query.getBpBankAcctUse() != null)
+		if (query.getBpBankAcctUses() != null)
```

Comment:
IMHO u shall also check if the list is empty

Reply:
Fixed, though based on https://github.com/metasfresh/metasfresh/pull/12281#discussion_r785075205 it would never be the case.